landingai/pipeline/frameset.py

```diff
@@ -98,6 +151,102 @@ def to_numpy_array(self, image_src: str = "") -> np.ndarray:
         img = self.image if image_src == "" else self.other_images[image_src]
         return np.asarray(img)
 
+    def show_image(
+        self,
+        image_src: str = "",
+        clear_nb_cell: bool = False,
+        *,
+        include_predictions: bool = False,
+    ) -> "Frame":
+        """Open an a window and display all the images.
+        Parameters
+        ----------
+        image_src (deprecated): if empty the source image will be displayed. Otherwise the image will be selected from `other_images`
```

Comment:
where is this other_images?

Reply:
This is a copy of the original comment. Since it is marked as deprecated (and it will not be documented the main documentation), I think we can fix it later (this `other_images` comment appears in several other places unfortunately).